### PR TITLE
Update virus-initiative-criteria.md

### PR DESCRIPTION
**Pull request by adamoboe**
Added the following text: The MVI Program has not currently accepting new applications for membership. Please contact MVI@microsoft.com for more information. We are pausing new members while we get out anti-cheat program up and running.

### CODE REVIEW

File: microsoft-365/security/intelligence/virus-initiative-criteria.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ The Microsoft Virus Initiative (MVI) helps organizations develop better-together
 
 ## Become a member
 
+The MVI Program has not currently accepting new applications for membership. Please contact MVI@microsoft.com for more information.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/intelligence/virus-initiative-criteria.md]:
```suggestion
> [!NOTE]
> The MVI Program is not currently accepting new applications for membership. Please contact MVI@microsoft.com for more information.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @adamoboe, kindly commit the suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@adamoboe Gentle reminder.

**Comment by yogkumgit** [comment]
@adamoboe Please update the suggested changes, in the event to no response, we may proceed to close this PR. Your cooperation is well appreciated. Thanks

**Pull request merged**